data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -70,52 +65,36 @@ public Object execute(String[] params) {
 
 	public void persist() {
 		Lineage lineage = new Lineage();
-
-//		LineageQueryInfo LineageQueryInfo = new LineageQueryInfo();
-//		LineageQueryInfo.setProcessId(122);
-//		LineageQueryInfo.setQueryId(UUID.randomUUID().toString());
-//		LineageQueryInfo.setQueryTypeId(1);
-//		LineageQueryInfo.setQueryString("select * from me");
-//		LineageQueryInfo.setInstanceExecId((long)1);
-//		LineageQueryInfo.setCreateTs(new Date());
-//		s.insert("call_procedures.InsertLineageQuery", lineageQueryInfo);
-		lineage.insertLineageQuery(lineageQueryInfo);
-
+		//inserting to LQ
+		//lineage.insertLineageQuery(lineageQueryInfo);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)